packages/kbn-es/README.mdx

```diff
@@ -28,6 +30,16 @@ Run from source with a configured data directory
 node scripts/es source --Epath.data=/home/me/es_data
 ```
 
+Run serverless with a specific image tag
+```
+node scripts/es serverless --tag git-fec36430fba2-x86_64
```

Comment:
Thoughts on something like `docker-default` and `docker-serverless` as the commands? Or via a `--serverless` flag similar to `node scripts/kibana --serverless`?

Reply:
@jbudz My original plan was to have the `--serverless` flag under a `docker` command, but that was getting messy at the time. The current iteration of the code it wouldn't be that bad though, just something like:

```ts
if(serverless) cluster.RunServerless(options)
else cluster.runDocker(options)
```

However, I think in the future that serverless could diverge enough that it's good to keep them split. If we wanted to add support for:

- [gradlew](https://github.com/elastic/elasticsearch-serverless#building-and-running-locally-with-docker) from source
- cloud clusters
- cloud data volumes

Open to combining the commands if you think this won't be necessary though.

Reply:
Works for me.